### PR TITLE
add hint about flyway usage

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description

Context: https://camunda.slack.com/archives/C0693F1NFK5/p1722411282734599

TL;DR: A customer was facing the issue that web modeler was not running as it claimed "the database schema is not empty" which can be due to functions being present in the database schema.

The newly added section should give the user (infra engineer) hints on what could be the problem and how it could be solved.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
DevEx will need more context to do a proper review. For now, we can only review for style and grammar.

**Comment by jonathanlukas** [comment]
@akeller thank you for flagging that. I added some context.

**Comment by christinaausley** [comment]
@jonathanlukas Are you just pending review from @wollefitz here, or any additional changes coming in?

**Comment by christinaausley** [comment]
@jonathanlukas Let me know when you're ready for final review and I'll have one more look 👍

**Comment by jonathanlukas** [comment]
@wollefitz the section has been moved and slightly renamed
@christinaausley this is ready for a final review I think

**Review by christinaausley — APPROVED** [review]
Nice 👍 Just uppercased Flyway.

**Review by wollefitz — APPROVED** [review]
Thanks for adding this!
